dbt_subprojects/daily_spellbook/models/eigenlayer/v2_withdrawal_queued_flattened.sql

```diff
@@ -0,0 +1,51 @@
+{{ 
+    config(
+        schema = 'eigenlayer',
+        alias = 'v2_withdrawal_queued_flattened',
+        post_hook='{{ expose_spells(\'["ethereum"]\',
+                                    "project",
+                                    "eigenlayer",
+                                    \'["bowenli"]\') }}'
+    )
+}}
```

Comment:
we should remove this given it's a staging view model, which will be used as a building block for later models. We'd only want the final models to be exposed as spells.

Reply:
this will be for both internal building block and external facing model, eg. to directly see who has queued withdraws

Reply:
all of our models will be both internal and external models